IB Diploma/cas_adviseStudents.php

```diff
@@ -36,152 +42,86 @@
         echo '<p>';
         echo "Your CAS staff role is $role. The students listed below are determined by your role, and student-staff relationship assignment.";
         echo '</p>';
```

Comment:
This could be added with a $form->setDescription(__m('')); so that it's tidied up and tagged for translation.

Reply:
Good call.